modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/search/EntriesChecker.java

```diff
@@ -195,6 +200,10 @@ private String _getRowCheckBox(
 		sb.append(name);
 		sb.append("\" title=\"");
 		sb.append(LanguageUtil.get(httpServletRequest.getLocale(), "select"));
+		sb.append("\" aria-label=\"");
+		sb.append(LanguageUtil.get(httpServletRequest.getLocale(), "select"));
```

Comment:
Concatenating messages limits the solution to those languages where "Select x" is translated in that same order. However, that is not the case for some languages where the `x` is in the beginning or even in the middle of the sentence, like the following ones:

```
Language_ar.properties:select-x=البحث في {0}
Language_de.properties:select-x={0} auswählen
Language_et.properties:select-x=Otsi {0}'e
Language_eu.properties:select-x={0}-an bilatu
Language_fa.properties:select-x=جست‌و‌جو در {0}
Language_hi_IN.properties:select-x={0} में सर्च
Language_iw.properties:select-x=בחר {0}
Language_ja.properties:select-x={0}を選択
Language_ko.properties:select-x={0} 선택
Language_lo.properties:select-x=ຄົ້ນ​ຫາ​ໃນ{0}
Language_nl.properties:select-x={0} selecteren
Language_ta_IN.properties:select-x={0} தேர்ந்தெடு
```
In addition, there's not always a blank space between the "select" and the "x":

```
Language_zh_CN.properties:select-x=选择{0}
Language_lo.properties:select-x=ຄົ້ນ​ຫາ​ໃນ{0}
```

For cases like this, rather than concatenating two messages, we need to rely on some other translation key (`select-x` looks an excellent candidate) so that `rowTitle` becomes a parameter of the language key we want to translate. This can be done using the `LanguageUtil.format()` API

Reply:
Thank you for the explanation. Done!